pkg/minikube/kubeconfig/kubeconfig.go

```diff
@@ -157,8 +157,8 @@ func configIssues(cfg *api.Config, contextName string, address string) []error {
 
 // populateCerts retains certs already defined in kubeconfig or sets default ones for those missing.
 func populateCerts(kcs *Settings, cfg api.Config, contextName string) {
-	lp := localpath.Profile(contextName)
-	gp := localpath.MiniPath()
+	gp := filepath.ToSlash(localpath.MiniPath())
```

Comment:
@bobsira I am curious have you explored or investigated that if we can just add  filepath.ToSlash to the "localpath" funcs ? would that break anything ?

Reply:
I have no exlopred this but I believe adding filepath.ToSlash inside the localpath functions  would introduce new mixed‑separator paths elsewhere and could re‑break the kubeconfig tests.